ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -406,6 +483,10 @@ export default defineComponent({
         return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
       })
 
+      // sort search results
+      const sortedResults = orderBy(localState.results, ['mhrNumber', 'ownerName.lastName', 'ownerName.middleName', 'ownerName.firsName'], ['asc', 'asc', 'asc', 'asc'])
```

Comment:
Why do we need to sort them? Isn't the backend sorting it for us?

Reply:
Backend sorts by owner name, I was sorting by mhrNum for ease of testing, removing

Reply:
Resolving this after discussion.